packages/plugin-core/src/components/lookup/utils.ts

```diff
@@ -864,6 +934,11 @@ export const filterPickerResults = ({
     );
   }
 
+  // Ending the query with a dot adds special processing of showing matched descendents.
+  if (transformedQuery.originalQuery.endsWith(".")) {
+    return sortForQueryEndingWithDot(transformedQuery, itemsToFilter);
```

Comment:
Probably best not to return but rather assign to `itemsToFilter` to have a single return in the method as well as allowing any other filtering (although none apply right now but for future) to take place.

Reply:
good point on the assignment. besides allowing for future filters, it also keeps the filter function simple (current implementation would introduce conditional logic that short circuits other processors)

unless there are any downsides i'm missing, would go for assignment

Reply:
Updated.